test/helpers/events.js

```diff
@@ -0,0 +1,15 @@
+export function dispatch (target, type, eventProperties) {
+	const event = new Event(type, { bubbles: true });
+	for (const property in eventProperties) {
```

Comment:
This fails on the linting rules currently:
```
/test/helpers/events.js:3:2 Error - The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype. (guard-for-in)
```

Reply:
Makes sense